### DISCUSSION TITLE
How can i have canva pro ?

### DISCUSSION THREAD

**Discussion by danaepenaguin** [community/community#34072] [Actions]
Hello to all,

I registered on Github with my school's student address because I saw on several articles that it was possible to have a canva pro account in their package.... Only I can't find the offer... Has it been removed... If anyone finds a solution, I really need a canva pro account and as a student I can't afford it... Thank you.

**Comment by Pralhad-Nasane** [comment]
The Canva Pro benefits are no longer offered in the GitHub Student Developer Pack.
